migrate/v0_11/migrate.go

```diff
@@ -57,11 +57,92 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
 		}
 		swaps = append(swaps, newSwap)
 	}
+
+	// -------------- ADD BTCB To BEP3 params --------------------
+	btcbAssetParam := v0_11bep3.NewAssetParam(
+		"btcb",
+		0,
+		v0_11bep3.SupplyLimit{
+			Limit:          sdk.NewInt(10000000000), // 100 BTC limit at launch
+			TimeLimited:    false,
+			TimePeriod:     time.Duration(0),
+			TimeBasedLimit: sdk.ZeroInt()},
+		true,
+		mustAccAddressFromBech32("kava14qsmvzprqvhwmgql9fr0u3zv9n2qla8zhnm5pc"),
+		v0_9Params.BnbDeputyFixedFee,
```

Comment:
The BnbDeputyFixedFee amount should vary by asset type in order to make them the same value. Let's use the existing 1000 bnb as the standard.